it.js

```diff
@@ -241,30 +241,30 @@ settings.crosshair.dot Punto
 settings.crosshair.opacity.use Usa opacità personalizzata
 settings.crosshair.opacity.custom Opacità personalizzata
 settings.crosshairm.header Crosshair (Throwing)
-settings.crosshairm.type Type
-settings.crosshairm.type.1 Shapes
-settings.crosshairm.type.2 Image
-settings.crosshairm.type.3 Precision
-settings.crosshairm.style Style
-settings.crosshairm.style.0 Cross
-settings.crosshairm.style.1 Hollow Circle
-settings.crosshairm.style.2 Solid Circle
-settings.crosshairm.style.3 Hollow Square
-settings.crosshairm.style.4 Solid Square
-settings.crosshairm.image Image
-settings.crosshairm.image.paste Crosshair Image
-settings.crosshairm.image.width Image Width
-settings.crosshairm.image.height Image Height
-settings.crosshairm.always Always Show
-settings.crosshairm.color Color
-settings.crosshairm.shadow.color Shadow Color
-settings.crosshairm.shadow.thickness Shadow Thickness
-settings.crosshairm.size Size
-settings.crosshairm.thickness Thickness
-settings.crosshairm.gap Gap
-settings.crosshairm.dot Dot
-settings.crosshairm.opacity.use Use Custom Opacity
-settings.crosshairm.opacity.custom Custom Opacity
+settings.crosshairm.type Tipo
+settings.crosshairm.type.1 Forma
+settings.crosshairm.type.2 Immagine
+settings.crosshairm.type.3 Precisione
+settings.crosshairm.style Stile
+settings.crosshairm.style.0 Croce
+settings.crosshairm.style.1 Cerchio con buco
+settings.crosshairm.style.2 Cerchio pieno
+settings.crosshairm.style.3 Quadrato con buco
+settings.crosshairm.style.4 Quadrato pieno
+settings.crosshairm.image Immagine
+settings.crosshairm.image.paste Immagine del crosshair
+settings.crosshairm.image.width Larghezza immagine
+settings.crosshairm.image.height Altezza immagine
+settings.crosshairm.always Sempre mostrato
```

Comment:
This was translated in "Mostra sempre" during previous translations (line 233), it would be best if kept that way.